src/api.js

```diff
@@ -1,21 +1,95 @@
 // src/api.js
 import { gql, GraphQLClient } from "graphql-request";
 
-// Base SubGraph
-// const endpoint = "https://i7n.app/gql";
-
-// Base Sepolia (Test) SubGraph
-// const endpoint = "https://api.i7n.dev/v1/graphql";
+// Hardcoded Endpoints
+const ENDPOINTS = {
+  railsMockApi: "http://localhost:3042/api/v1/graph",
+  baseSepolia: "https://api.i7n.dev/v1/graphql",
+  base: "https://i7n.app/gql",
+};
 
-// Rails Mock API
-const endpoint = "http://localhost:3042/api/v1/graph";
+// Select which endpoint to use
+// Change this to switch endpoints:
+//'railsMockApi' | 'baseSepolia' | 'base'
+const data_endpoint = "railsMockApi";
 
-const client = new GraphQLClient(endpoint);
+// Create GraphQL client with selected endpoint
+const client = new GraphQLClient(ENDPOINTS[data_endpoint]);
 
 export const fetchTriples = async () => {
+  let query, data;
+  switch (data_endpoint) {
+    case "base":
+      query = gql`
+        query {
+          triples(limit: 1000) {
+            items {
+              id
+              subject {
+                label
+                id
+              }
+              predicate {
+                label
+                id
+              }
+              object {
+                label
+                id
+              }
+            }
+          }
+        }
+      `;
+      data = await client.request(query);
+      return data.triples.items;
+    default:
+      query = gql`
+        query {
+          triples(limit: 1000) {
+            id
+            subject {
+              label
+              id
+            }
+            predicate {
+              label
+              id
+            }
+            object {
+              label
+              id
+            }
+          }
+        }
+      `;
+      data = await client.request(query);
+      return data.triples;
+  }
+};
+
+// Export current endpoint for potential use in other components
+export const getCurrentEndpoint = () => ENDPOINTS[data_endpoint];
+
+export const fetchTriplesForNode = async (nodeId) => {
   const query = gql`
     query {
-      triples(limit: 500) {
+      triples(filter: { subject: "${nodeId}" }) {
```

Comment:
This will only show related triple where the clicked node is the subject. 

A better approach would be to query this cliked atom and get the list of all the triples related to it by a similar gql request : 
https://studio.apollographql.com/sandbox/explorer?endpoint=https%3A%2F%2Fapi.i7n.dev%2Fv1%2Fgraphql&explorerURLState=N4IgJg9gxgrgtgUwHYBcQC4QEcYIE4CeABAIIoRwAUAJAIblwCSY6RS8%2BAllAIQCURYAB0kRIvQqVOLInQbMBw0WJVjaAZwDKMAEYArBFBSCRqsQBtaOhOdNnpdsQF9HRVxoDy%2Bw8aVmilta2yqoOIUQu4RoACngIYNz0CCbhFlY2rmJhqpF2kZEgADQgAG60eJxW5gjqGCDCeHZCIBJMYM2sAIwATEKNSE4gTkA